src/ontology/mondo-edit.obo

```diff
@@ -546354,7 +546380,7 @@ synonym: "IPF" EXACT ABBREVIATION [MONDO:Lexical, OMIM:178500]
 synonym: "pulmonary fibrosis, idiopathic" RELATED [MONDO:Lexical, OMIM:178500]
 synonym: "UIP" EXACT ABBREVIATION [Orphanet:2032]
 synonym: "usual interstitial pneumonia" EXACT [Orphanet:2032]
-xref: DOID:0050156 {source="MONDO:equivalentTo", source="EFO:0000768"}
```

Comment:
DOID:0050156 is still annotated as the source for some of the synonyms, xrefs and superclass assertions on this term:

synonym: "cryptogenic fibrosing alveolitis" EXACT [DOID:0050156, Orphanet:2032]
synonym: "fibrocystic pulmonary dysplasia" EXACT [DOID:0050156, OMIM:178500]
synonym: "idiopathic pulmonary fibrosis, familial" EXACT [DOID:0050156, OMIM:178500]
xref: ICD10CM:J84.112 {source="DOID:0050156", source="MONDO:equivalentTo"}
xref: MESH:D054990 {source="DOID:0050156", source="Orphanet:2032/e", source="MONDO:equivalentTo", source="EFO:0000768", source="Orphanet:2032"}
xref: NCIT:C35715 {source="DOID:0050156"}
xref: NCIT:C35716 {source="DOID:0050156", source="MONDO:equivalentTo", source="EFO:0000768"}
xref: OMIM:178500 {source="DOID:0050156", source="Orphanet:2032/e", source="MONDO:equivalentTo", source="EFO:0000768", source="Orphanet:2032"}
xref: SCTID:196125002 {source="DOID:0050156", source="MONDO:equivalentTo"}
xref: SCTID:233721005 {source="DOID:0050156"}
xref: SCTID:237121003 {source="DOID:0050156"}
xref: SCTID:28168000 {source="DOID:0050156"}
xref: SCTID:426437004 {source="DOID:0050156"}
is_a: MONDO:0002429 {source="DOID:0050156", source="MESH:D054990", source="MONDO:0008345", source="Orphanet:2032", source="https://en.wikipedia.org/wiki/Idiopathic_interstitial_pneumonia"} ! idiopathic interstitial pneumonia
is_a: MONDO:0002771 {source="DOID:0050156", source="MESH:D054990", source="NCIT:C35716"} ! pulmonary fibrosis

I think this should be reviewed.

Should DOID:0050156 be an xref on another term in Mondo?

Reply:
The reason why I did not add this DOID to another term is because it requires much more review. 
This term came from a split (https://github.com/monarch-initiative/mondo/issues/4508)
It is defined as being idiopathic AND due to a variation in a gene. I need to investigate deeper to figure out what is going on there. Therefore I did not reassign DOID:0050156 to another Mondo term. I am planning on reviewing for the next release. 

I removed this DO id as a source for synonyms, but the synonyms are still supported by other sources. So this absolutely requires further investigations.

Reply:
sounds good!